Dockerfiles/midstream/run_tests.py

```diff
@@ -42,18 +42,26 @@ def run_validate_operator_bundle(self):
                     -e 'work_dir={work_dir}'".format(operator_dir=self.operator_dir,
                                                     operator_work_dir=self.operator_work_dir,
                                                     work_dir=self.work_dir)
-        result = subprocess.run(exec_cmd, shell=True)
+        result = subprocess.run(exec_cmd, shell=True, capture_output=True)
         return result
 
     def test_for_extract_and_validate_bundle_image(self):
         result = self.run_extract_operator_bundle()
-        self.assertEqual(0,
-                         result.returncode,
-                         "extract-operator-bundle.yml failed")
+        if(50 <= result.returncode <= 99):
```

Comment:
ansible playbooks does not return such error codes.

This scrpt `run_test.py` has wto retun such error codes.